ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js

```diff
@@ -1,10 +1,261 @@
+        return {
+          en: 'Get Under Wings',
+        };
+      },
+    },
+    {
+      id: 'Puppet Aegis Refraction Cannons 2',
+      netRegex: NetRegexes.startsUsing({ source: '813P-Operated Aegis Unit', id: '507F', capture: false }),
+      alertText: function(data) {
+        return {
+          en: 'Get Between Wings',
+        };
+      },
+    },
+    {
+      id: 'Puppet Aegis High-Powered Laser',
+      // This is also head marker 003E, but since there's three stacks, just say "stack".
+      netRegex: NetRegexes.startsUsing({ source: '813P-Operated Aegis Unit', id: '508F', capture: false }),
+      response: Responses.stack(),
+    },
+    {
+      id: 'Puppet Aegis Life\'s Last Song',
+      netRegex: NetRegexes.startsUsing({ source: '813P-Operated Aegis Unit', id: '53B3', capture: false }),
+      // This is more a "if you haven't done this ever or in a while, here's a reminder."
+      // Tell it once, but as this repeats nearly continously forever, only say it once.
+      suppressSeconds: 9999,
+      infoText: {
+        en: 'Dodge into ring gap',
+      },
+    },
+    {
+      id: 'Puppet Light Volt Array',
+      netRegex: NetRegexes.startsUsing({ source: 'Light Artillery Unit', id: '5211' }),
+      condition: (data) => data.CanSilence(),
+      // Multiple of these cast at the same time.
+      suppressSeconds: 5,
+      response: Responses.interrupt('alarm'),
+    },
+    {
+      id: 'Puppet Light Homing Missile',
+      netRegex: NetRegexes.headMarker({ id: '008B' }),
+      condition: Conditions.targetIsYou(),
+      alertText: {
+        en: 'Fire Puddle on YOU',
+      },
+    },
+    {
+      id: 'Puppet Superior Shield Protocol',
+      netRegex: NetRegexes.startsUsing({ id: '4FA[678]', capture: false }),
+      run: (data) => data.phase = 'superior',
+    },
+    {
+      id: 'Puppet Superior Missile Command',
+      netRegex: NetRegexes.startsUsing({ id: '4FBD', capture: false }),
+      suppressSeconds: 5,
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      // This is for Maneuver: Incendiary Bombing and Maneuver: Area Bombardment.
+      id: 'Puppet Superior Incendiary Bombing',
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: (data, matches) => data.me === matches.target && data.phase === 'superior',
+      alertText: {
+        en: 'Fire Puddle on YOU',
+      },
+    },
+    {
+      id: 'Puppet Superior High-Powered Laser',
+      // Note: no 1B marker for this???
+      netRegex: NetRegexes.startsUsing({ id: '4FB4', capture: false }),
+      suppressSeconds: 5,
+      alertText: {
+        en: 'Line Stack (Point Outside)',
+      },
+    },
+    {
+      // TODO: we could theoretically get combatants here when sharp turn starts casting.
+      // Sharp Turn ability ids differentiate left/right.  We could use effect ids to know group.
+      // But how would you communicate where to stand? "behind and slightly/very right"?
```

Comment:
I believe it's only three possibilities, right? Viewing the arena as an equilateral triangle with the single point as South, we have "stand left of point", "stand south of point",  and "stand right of point". Since the bosses seem to face counterclockwise, we could call it as "behind boss", "full (south|northwest|northeast)", "boss right flank".

Probably not worth doing for this release, but I think it's doable.